examples/README.md

```diff
@@ -890,21 +890,18 @@ The data output of the workflow contains the information of the exception caught
           }
        }
     ],
-    "stateDataFilter": {
-       "dataOutputPath": "${ .exceptions }"
-    },
     "transition": "ApplyOrder",
     "onErrors": [
        {
-         "error": "Missing order id",
+         "condition": "${ .exceptions | .[] | .error == 'Missing order id' }",
```

Comment:
I assume your condition has to evaluate to true or false here (which we should also document in the specification.md).

Also this assumes (which should prob also be documented) that runtimes have to provide and inject an 'exceptions' json array into the state if error happens which has the format

```json
{
    "exceptions": [
        {
            "error": "...",
            "code": "..."
        },
        ...
    ]
    
}
```

In your example the jq should be "any(.exceptions[]; .error == "Missing order id")" which will return you true/false if an item in the exceptions array exists that has the error property set to "missing order id".